.circleci/config.yml

```diff
@@ -11,20 +11,33 @@ jobs:
             BLENDER_VERSION: 2.79
             GLIBC_VERSION: 219
            command: |
-             curl -Ls http://download.blender.org/release/Blender$BLENDER_MAJOR/blender-$BLENDER_VERSION-linux-glibc$GLIBC_VERSION-x86_64.tar.bz2 | tar -xjv
-             sudo mv blender-${BLENDER_VERSION}-linux-glibc${GLIBC_VERSION}-x86_64 /usr/local/bin/blender
+             curl -Ls http://download.blender.org/release/Blender$BLENDER_MAJOR/blender-$BLENDER_VERSION-linux-glibc$GLIBC_VERSION-x86_64.tar.bz2 | sudo tar -xjv -C /opt/
+             sudo ln -s /opt/blender-${BLENDER_VERSION}-linux-glibc${GLIBC_VERSION}-x86_64/blender /usr/local/bin/blender
        - run:
-           name: Install Python3.6 and pip
+           name: Install Python3.6
            command: |
              sudo apt-get update
-             sudo apt-get install software-properties-common
-             sudo add-apt-repository -y ppa:jonathonf/python-3.6
+             sudo apt-get install software-properties-common libglu1
+             sudo add-apt-repository -y ppa:deadsnakes/ppa
              sudo apt-get update
-             sudo apt-get install python3.6 python3.6-dev
-             wget https://bootstrap.pypa.io/get-pip.py
-             sudo python3.6 get-pip.py
+             sudo apt-get install python3.6 python3.6-dev python3.6-venv
+       - run:
+           name: Prepare venv with requirements
+           command: |
+             python3.6 -m venv .venv
+             . .venv/bin/activate
+             pip install -r requirements.txt
+       - run:
+           name: Check versions
+           command: |
+             . .venv/bin/activate
```

Comment:
I think it's not required to activate venv every time

Reply:
i thought so too, so tried it first :)

circleci proved me wrong, you see it at the versions step
https://circleci.com/gh/golemfactory/blenderapp/52